### PR TITLE
feat:(checkbox) migrate checkbox component

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Migration of checkbox component. 

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#2386](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2386)

**How to test**  
_Add description how to test if possible_
1. Go to the storybook link below
2. Check in components -> checkbox
3. Check that the design aligns with the one in figma.

**Screenshots**  
-

**Additional context**  
-

### CODE REVIEW

**Comment by theJohnnyMe** [comment]
For me, there are three things to fix:

- Label shows on the left side instead on right compared to the old storybook
- Label in the disabled story is not disabled
- "With Label" story seems redundant

**Comment by mJarsater** [comment]
> For me, there are three things to fix:
> 
> * Label shows on the left side instead on right compared to the old storybook
> * Label in the disabled story is not disabled
> * "With Label" story seems redundant

@theJohnnyMe Thanks!

File: components/src/components/checkbox/checkbox.stories.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      type: 'boolean',
+      table: {
+        disable: true,
+      },
+    },
+  },
+};
+
+const Template = args => {
+  console.log(args);
```
Comment (alexandereneroth) [inline components/src/components/checkbox/checkbox.stories.ts:41]:
```suggestion
```

**Review by alexandereneroth — APPROVED** [review]
Nice!

**Pull request merged**